apps/src/p5lab/AnimationPicker/AnimationPickerListItem.jsx

```diff
@@ -18,6 +18,7 @@ export default class AnimationPickerListItem extends React.Component {
     category: PropTypes.string,
     selected: PropTypes.bool,
     isBackgroundsTab: PropTypes.bool,
+    isAnimationJsonMode: PropTypes.bool,
```

Comment:
Added this prop so that we could display an image tile for the 'All' category that is more accurate  on the Generate Animation JSON levelbuilder tool page.